WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Deposits Overview/WooPaymentsDepositsCurrencyOverviewView.swift

```diff
@@ -103,7 +88,10 @@ struct WooPaymentsDepositsCurrencyOverviewView: View {
                             .multilineTextAlignment(.leading)
                     }
                 }
+                .padding(.top)
                 .padding(.bottom)
+
+                Divider()
```

Comment:
Consider moving this to a separate view and applying the horizontal padding to the content above only, not the whole view (line 100). That way, the separator can be full width.

Reply:
I went ahead with moving the divider to the parent view instead for it to be full width: https://github.com/woocommerce/woocommerce-ios/pull/11533/commits/77b5dab111db75dadb3b62b617823a291c517cb3